src/state/api.js

```diff
@@ -58,24 +58,34 @@ export async function getProbeData(params, token) {
 export function getSearchResults(queryString, resultsLimit) {
   const getFormattedSearchURL = (str, product = 'desktop') => {
     const URLResult = new URL('__BASE_SEARCH_DOMAIN__');
+    const strFragments = str.split(/\s+/);
+
     const params = new URLSearchParams();
     const queryOptions = [];
     const stringScalars =
-      '(type.eq.scalar,definition->details->>kind.eq.string)';
+      '(type.eq.scalar,info->calculated->latest_history->details->>kind.eq.string)';
 
     queryOptions.push(`name.eq.${str}`);
-    queryOptions.push(`index.phfts(english).${str}`);
+    queryOptions.push(`search.phfts(simple).${str}`);
```

Comment:
`search.plfts(simple)` -- It's the difference of a phrase query or plain query. I think plain will be better suited here.